src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -1949,10 +2026,34 @@ static void window_tile_inspector_paint(rct_window* w, rct_drawpixelinfo* dpi)
                 screenCoords.y = w->windowPos.y + w->widgets[WIDX_TRACK_SPINNER_HEIGHT].top;
                 gfx_draw_string_left(dpi, STR_TILE_INSPECTOR_BASE_HEIGHT_FULL, nullptr, COLOUR_WHITE, screenCoords);
 
+                // brakes/booster speed label
+                if (tileElement->AsTrack())
+                {
+                    if (tileElement->AsTrack()->GetTrackType() == TRACK_ELEM_BRAKES
+                        || tileElement->AsTrack()->GetTrackType() == TRACK_ELEM_BOOSTER)
+                    {
+                        screenCoords.y = w->windowPos.y + w->widgets[WIDX_TRACK_SPINNER_SPEED].top;
+                        gfx_draw_string_left(dpi, STR_TILE_INSPECTOR_TRACK_SPEED, nullptr, COLOUR_WHITE, screenCoords);
+
+                        // brakes/booster speed spinner value
+                        screenCoords.x = w->windowPos.x + w->widgets[WIDX_TRACK_SPINNER_SPEED].left + 3;
+                        screenCoords.y = w->windowPos.y + w->widgets[WIDX_TRACK_SPINNER_SPEED].top;
+                        int32_t speed = static_cast<int32_t>(tileElement->AsTrack()->GetBrakeBoosterSpeed());
+
+                        // convert the speed to a readable format
+                        auto ft = Formatter();
+                        ft.Add<int32_t>(speed);
+
+                        DrawTextBasic(dpi, screenCoords, STR_FORMAT_INTEGER, ft, COLOUR_WHITE);
+                    }
+                }
```

Comment:
Following with the above comment, this needs adjusting as well. Simply draw a "-" inside the disabled widget here.